moveit_ros/hybrid_planning/global_planner/global_planner_plugins/src/moveit_planning_pipeline.cpp

```diff
@@ -97,44 +97,86 @@ moveit_msgs::msg::MotionPlanResponse MoveItPlanningPipeline::plan(
     return response;
   }
 
-  // Process goal
-  if ((global_goal_handle->get_goal())->motion_sequence.items.size() > 1)
+  // If multiple waypoints were requested, assume we have a fully planned trajectory already.
+  // Just forward to the local planner.
+  // This works only for a JointConstraint type.
+  size_t num_waypoints = global_goal_handle->get_goal()->motion_sequence.items.size();
+  if (num_waypoints > 1)
   {
-    RCLCPP_WARN(LOGGER, "Global planner received motion sequence request with more than one item but the "
```

Comment:
In case this behavior is not configurable, I think it might be useful to notify the user that the global planning request was not processed by any planner

Reply:
It's configurable now.  :+1: